src/logging.h

```diff
@@ -75,6 +84,7 @@ namespace BCLog {
         std::atomic<uint32_t> m_categories{0};
 
         std::string LogTimestampStr(const std::string& str);
+        std::string Prefix(const std::string& str);
```

Comment:
Perhaps this shouldn't be a public method; I wasn't sure if it may be generally useful.